### PR TITLE
Refactor/results

### LINKED ISSUE #20: Create Results component with all search results
- [x] Retrieve data from getCoordinates API based on the query entered in search bar
- [x] Render the Results component after button click
- [x] Results component will render cards for each result returned
- [x] Handle errors!!!
- [x] Style
- [x] Clear input upon search button clicked

### LINKED ISSUE #35: target=blank research/bug
Find bug in React that doesn't like target=blank for outside links

add `rel="noopener noreferrer"` in all a tags where `target=_blank` is

### PR DESCRIPTION
**Pull request by ErinUntermeyer**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- Clears input upon clicking search
- Fixes bug with unique key in Home when rendering Card
- Fixes lint errors for target: _blank
- Closes #35 closes #20 

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes
- Console is free of errors as well as linter

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/About/About.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import './About.scss'
-import React, { useState, useEffect } from "react";
+import React from "react";
```
Comment (JoshSevy) [inline src/About/About.tsx:2]: Thank you for going through and removing those linter errors

File: src/Home/Home.tsx [hunk 2]
```diff
@@ -38,7 +40,8 @@ const Home: React.FC = () => {
   </a>,
   <a
    href="https://www.active.com/outdoors/articles/5-tips-for-successful-altitude-acclimation"
-   target="_blank"
+	 target="_blank"
+	 rel="noopener noreferrer"
```
Comment (JoshSevy) [inline src/Home/Home.tsx:44]: This is good to know, thanks for bringing it up in chat.

File: src/About/About.tsx (hunk 1 above)
Comment (leighlars) [reply src/About/About.tsx:2]: <3

File: src/Home/Home.tsx (hunk 2 above)
Comment (leighlars) [reply src/Home/Home.tsx:44]: thank you!!

**Comment by leighlars** [comment]
You rock!! Thank you.

**Pull request merged**